lib/livebook/session.ex

```diff
@@ -976,6 +984,13 @@ defmodule Livebook.Session do
     {:noreply, handle_operation(state, operation)}
   end
 
+  def handle_cast({:put_env_vars, _client_pid, env_vars}, state) do
+    env_vars = Enum.map(env_vars, &{&1.key, &1.value})
+    Livebook.Runtime.put_system_envs(state.data.runtime, env_vars)
```

Comment:
We will need a separate function for deleting env vars, and then we need more granular messages to know if we should set or delete.

(Currently secrets can't be deleted, that's why we don't have such function yet)